src/main/java/com/karankumar/booksapi/repository/BookRepository.java

```diff
@@ -24,5 +24,7 @@
   
     @Query("SELECT DISTINCT b FROM Book b LEFT JOIN FETCH b.authors")
     List<Book> findAllBooks();
-  
+
+    @Query("SELECT DISTINCT b FROM Book b LEFT JOIN FETCH b.authors WHERE b.isbn13=:isbn")
+    Book findBookByISBN(String isbn);
```

Comment:
Could you please remove the JPQL query?

Reply:
Not sure what you mean could you elaborate?

Reply:
This line:
```java
@Query("SELECT DISTINCT b FROM Book b LEFT JOIN FETCH b.authors WHERE b.isbn13=:isbn") 
```